composer.json

```
@@ -20,17 +20,17 @@
     ],
     "homepage": "https://docs.sonata-project.org/projects/SonataSeoBundle",
     "require": {
-        "php": "^8.0",
+        "php": "^8.0|^8.2",
```

```suggestion
        "php": "^8.0",
```
Is ok, it means already 8.0, 8.1, 8.2, 8.3 etc

---

Okay @VincentLanglet noted.